controllers/suite_util_test.go

```diff
@@ -311,6 +311,17 @@ func waitDoneMessageOnComponent(componentKey types.NamespacedName) {
 }
 
 func expectPacBuildStatus(componentKey types.NamespacedName, state string, errID int, errMessage string, mergeURL string) {
+	// should first wait for desired state, as there is small time window when
+	// message might be still done from previous run, but even though action already finished, it didn't update status yet
```

Comment:
> message might be still done from previous run

I believe this should never happen as we recreate the Component in each test case.
If you mean checking Component state within the same test case, please make the message clearer (at least replace run with check).

Reply:
rephrased